src/examples/delete.yml

```diff
@@ -0,0 +1,21 @@
+description: |
+  Delete kubernetes resource.
+
+usage:
+  version: 2.1
+
+  orbs:
+    kube-orb: circleci/kubernetes@x.y.z
+
+  jobs:
+    build:
+      machine:
```

Comment:
As the example doesn't rely on minikube, I think there's no special reason to use the machine executor. Could we use the docker executor instead?

For example: 
```
docker:
      - image: cimg/base:stable
```

Reply:
I've fixed this in https://github.com/CircleCI-Public/kubernetes-orb/pull/26/commits/5902ad985f23ddd77834390600923a5d15e418c1 gave the job a more descriptive name too